scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/JobsController.java

```diff
@@ -260,6 +267,49 @@ public void onFailure(Throwable caught) {
         });
     }
 
+    /**
+     * Gets the signals of a job
+     *
+     * @param jobId id of the job
+     * @param menu the menu of a job
+     * @param jobsListGrid
+     */
+    public void getJobSignals(String jobId, Menu menu, JobsListGrid jobsListGrid) {
```

Comment:
why do u pass menu ? u are not using it in the controller ?

Reply:
I refactored it as class variable